File: Jumping on the clouds.py
def jumpingOnClouds(c):
    current = 0
    end = len(c) - 1
    jumps = 0

    while current < end:
        if ((current + 2) <= end) and (c[current + 2] == 0):
            current += 2
            jumps += 1
        elif c[current + 1] == 0:
            current += 1
            jumps += 1

    return jumps

File: Array exercises/Jumping on the clouds.java
public static int jumpingOnClouds(int[] c) {
        int n = c.length;
        int total_jumps = 0;

        int i=0;
        while(i<n-1){
            if(i+2 < n && c[i+2] != 1){
                total_jumps++;
                i=i+2;
            }
            else{
                total_jumps++;
                i++;
            }
        }
    return total_jumps;
}
